README.md

```diff
@@ -36,19 +36,18 @@ import (
 )
 
 func main() {
-	cli := airflow.NewAPIClient(&airflow.Configuration{
-		Scheme:   "http",
-		Host:     "localhost:8080",
-		BasePath: "/api/v1",
-	})
+	conf := airflow.NewConfiguration()
+	conf.Host = "localhost:8080"
+	conf.Scheme = "http"
+	cli := airflow.NewAPIClient(conf)
 
 	cred := airflow.BasicAuth{
 		UserName: "username",
 		Password: "password",
 	}
 	ctx := context.WithValue(context.Background(), airflow.ContextBasicAuth, cred)
 
-	variable, _, err := cli.VariableApi.GetVariable(ctx, "foo")
+	variable, _, err := cli.VariableApi.GetVariable(ctx, "foo").Execute()
```

Comment:
unfortunately openapi-generator 5.x introduced breaking change in their go code gen.